aal_purchase_form/models/purchase.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from openerp import models, api
+from num2words import num2words
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.multi
+    def amount_text(self, amount):
```

Comment:
```suggestion
    def amount_text(self, amount):
```
delete function is not used